code/model/UserDefinedForm.php

```diff
@@ -165,11 +165,12 @@ public function getCMSFields() {
 
 			// get a list of all field names and values used for print and export CSV views of the GridField below.
 			$columnSQL = <<<SQL
-SELECT "Name", "Title"
+SELECT "SubmittedFormField"."Name" as "Name", "SubmittedFormField"."Title" as "Title", IFNULL("EditableFormField"."Sort", 999) AS "Sort"
```

Comment:
I believe that `COALESCE()` is the ANSI-SQL equivalent of IFNULL, which would preserve cross-database support.